packages/app-mobile/components/screens/NoteTagsDialog.js

```diff
@@ -170,10 +175,23 @@ class NoteTagsDialogComponent extends React.Component {
 						onChangeText={value => {
 							this.setState({ newTags: value });
 						}}
-						style={this.styles().newTagBoxInput}
+						style={this.styles().tagBoxInput}
+						placeholder="e.g. foo,bar"
```

Comment:
I added a placeholder here to help user understand that they can actually input a comma-seperated list of tags, which is not very self explanatory. But I'm OK to keep it as-is.

Reply:
Ok I think that helps, but "foo,bar" I think is hard to translate. Could you make it instead `_('tag1, tag2...')`?